sklearn/tests/test_common.py

```diff
@@ -358,3 +362,25 @@ def test_transformer_n_iter():
 
         if hasattr(estimator, "max_iter") and name not in external_solver:
             yield check_transformer_n_iter, name, estimator
+
+
+def test_partial_fit():
+    """Tests all estimators which support partial_fit"""
+
+    skiplist = ("HashingVectorizer",
+                "MiniBatchDictionaryLearning",
```

Comment:
They don't... sorry... I guess I had misunderstood something there too... ( like in `MiniBatchKMeans`... unable to figure what though... ) have removed them from this list...